### DISCUSSION TITLE
No "Client Service" endpoint?

### DISCUSSION THREAD

**Discussion by panday1995** [hootnot/saxo_openapi#24] [General]
## Description

I am trying to use saxo_openapi to access "Trading Conditions - Cost", but there is no such endpoint in the list. Is it going to be added? 

I want to get total costs for pre trading, as well as its breakdown, such as spread cost, commission cost  and conversion cost.

**Comment by hootnot** [comment]
"Trading Conditions - Cost" ? I don't see that one in the SAXO docs. 

This API wrapper only covers endpoints of the SAXO API. But it is SAXO Bank that is responsible for the API if it comes to development and deployment.
It sounds like you want additional functionality in the API. Then you need to contact SAXO.

If it is some endpoint in the API that is not covered by this API-wrapper, then please be specific

**Reply by panday1995** [reply]
Hi, 

AFAIK It is not cover in this API wrapper. Please see this: https://www.developer.saxo/openapi/referencedocs/cs/v1/tradingconditions-cost

**Reply by hootnot** [reply]
Yes. That clarifies. That whole group (Client Services) is not covered (yet).
